sigma/plugins/utility/other/define.py

```diff
@@ -0,0 +1,78 @@
+import discord
+import aiohttp
+from bs4 import BeautifulSoup
+
+async def define(cmd, message, args):
+
+    definition = ""
+    defNum = 1
+    hasPrinted = False
```

Comment:
If you don't predefine, the default result of `if hasPrinted` is false, so this is redundent